CMSgov/ato-blueprint
# Feature/pre commit controls

Jira: https://jiraent.cms.gov/browse/ISPGBSS-800

QA:

- Pull down the branch
- Run `python run.py dev`
- Enter the container: `docker exec -it govready-q-dev bash`
- Run the tests `python3 manage.py test controls/`
- You should see:
`Ran 40 tests in [SOME SECONDS]s

OK (skipped=5)`
